src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -181,12 +181,7 @@ class ExposureSubmissionCoordinator: NSObject, ExposureSubmissionCoordinating, R
 				self?.model.exposureSubmissionService.loadSupportedCountries(
 					isLoading: isLoading,
 					onSuccess: { supportedCountries in
-						self?.showTestResultSubmissionConsentScreen(
-							supportedCountries: supportedCountries,
-							onDismiss: {
-								self?.showTestResultAvailableCloseAlert()
-							}
-						)
+						self?.showTestResultSubmissionConsentScreen(supportedCountries: supportedCountries, onDismiss: nil)
```

Comment:
please keep the line break style
```suggestion
self?.showTestResultSubmissionConsentScreen(
 							supportedCountries: supportedCountries,
 							onDismiss: nil
 						)						self?.showTestResultSubmissionConsentScreen(supportedCountries: supportedCountries, onDismiss: nil)
```

Reply:
Ok sure